wagtailpolls/templates/wagtailpolls/poll_list.html

```diff
@@ -35,13 +35,13 @@
 			</tbody>
 </table>
 <div class="pagination">
-	<p>Page {{page.number}} of {{paginator.num_pages}}</p>
+	<p>{% blocktrans %}Page{% endblocktrans %} {{page.number}} {% blocktrans %}of{% endblocktrans %} {{paginator.num_pages}}</p>
```

Comment:
Same comment as above regarding ordering of sentences like this. Something like:

```html+django
<p>{% blocktrans with number=page.number total=paginator.num_pages %}Page {{ number }} of {{ total }}{% endblock %}</p>
```